tools/rosbag/src/rosbag/rosbag_main.py

```diff
@@ -143,6 +149,12 @@ def record_cmd(argv):
         signal.SIGTERM,
         lambda signum, frame: _stop_process(signum, frame, old_handler, process)
     )
+
+    old_handler = signal.signal(
+        signal.SIGINT,
+        lambda signum, frame: _send_process_sigint(signum, frame, old_handler, process)
+    )
+
```

Comment:
The `SIGINT` handling in Python seems to be orthogonal to the ticket title. Can you please clarify how this relates to the described use case. Maybe it should be split into a separate PR if it addresses a different problem.

Reply:
This is fixing the behavior when sending `kill -2 $(pgrep rosbag)` , outlined here: https://github.com/ros/ros_comm/pull/2038#issuecomment-691362372

Reply:
Please update the ticket title to reflect this.